src/components/molecules/Common/Notification/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+
+import Flex from "@reearth/components/atoms/Flex";
+import Icon from "@reearth/components/atoms/Icon";
+import Text from "@reearth/components/atoms/Text";
+import { styled, metrics, useTheme } from "@reearth/theme";
+
+export type NotificationStyleType = "error" | "warning" | "info" | "success";
+export type NotificationType = {
+  type: "error" | "warning" | "info" | "success";
```

Comment:
```suggestion
  type: NotificationStyleType;
```

Reply:
NotificationType comes from `@reearth/state` and we have it set up so I can't import from there. I thought about exporting it from `organisms/NotificationBanner` but then I'd have looping imports which isn't good.
Any other options or can I leave this the way it is?

Reply:
You can leave it. In thi case, it is not a problem to have two types. If one of the types is changed and the types are no longer aligned, an error will occur and you'll know immediately.